apps/fluent-tester/wdio.conf.macos.js

```diff
@@ -113,8 +113,22 @@ exports.config = {
    * Hook that gets executed before the suite starts
    * @param {Object} suite suite details
    */
-  // beforeSuite: function (suite) {
-  // },
+  beforeSuite: function () {
+    // Maximize app window
+    let fluentTesterWindow = null;
+    browser.waitUntil(() => {
+      fluentTesterWindow = $('//*[@title="Fluent Tester" and @elementType=4]');
+      return fluentTesterWindow != null;
+    },
+    {
+      timeout: 10000,
+      timeoutMsg: 'Could not find the FluentTester window. Cannot maximize app.',
+      interval: 500
+    });
+
+    const maxButton = fluentTesterWindow.$('//*[@identifier="_XCUI:FullScreenWindow" and @elementType=9]');
+    maxButton.click();
```

Comment:
Could we leave a more detailed comment to the tune of "appium doesn't have an easy way to maximize the test app window, so we look up the maximize window button directly through it's XCUI identifier"?

Reply:
Improved comment in newest iteration